src/WpfMath/TexFormulaParser.cs

```diff
@@ -355,32 +356,63 @@ internal static SourceSpan ReadElementGroup(SourceSpan value, ref int position,
             if (value[position] != openChar)
                 return null;
 
-            return ReadElementGroup(value, ref position, openChar, closeChar);
+            return ReadArgumentGroup(value, ref position, openChar, closeChar);
         }
 
-        /// <summary>Reads an element: typically, a curly brace-enclosed value group or a singular value.</summary>
+        /// <summary>Reads an argument: typically, a curly brace-enclosed value group, a singular value or an escaped sequence of letters/a character.</summary>
         /// <exception cref="TexParseException">Will be thrown for ill-formed groups.</exception>
-        internal static SourceSpan ReadElement(SourceSpan value, ref int position)
+        internal static SourceSpan ReadArgument(SourceSpan value, ref int position)
         {
-            SkipWhiteSpace(value, ref position);
-
-            if (position == value.Length)
-                throw new TexParseException("An element is missing");
-
-            if (value[position] == leftGroupChar)
+            if (position < value.Length)
             {
-                return ReadElementGroup(value, ref position, leftGroupChar, rightGroupChar);
-            }
+                if (value[position] == leftGroupChar)
+                    return ReadArgumentGroup(value, ref position, leftGroupChar, rightGroupChar);
+                else if (value[position] == escapeChar)
```

Comment:
I suggest we extract a method to read the named stuff after the `\` and use the method here.